lighthouse-core/test/gather/computed/trace-of-tab-test.js

```diff
@@ -16,6 +16,8 @@ const preactTrace = require('../../fixtures/traces/preactjs.com_ts_of_undefined.
 const noFMPtrace = require('../../fixtures/traces/no_fmp_event.json');
 const noFCPtrace = require('../../fixtures/traces/airhorner_no_fcp');
 const backgroundTabTrace = require('../../fixtures/traces/backgrounded-tab-missing-paints');
+const tracingStartedInBrowserTrace =
+  require('../../fixtures/traces/tracing-started-in-browser.json');
```

Comment:
this whole fixture is just to exercise that one event right?

could we use a reduced trace or even just a fake array of events for this? 6k lines for the 1 relevant event is a lot :)